src/installer/managed/Microsoft.NET.HostModel/Bundle/Codesign.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.CoreSetup
+{
+    public class Codesign
+    {
+        private const string CodesignPath = @"/usr/bin/codesign";
+
+        public static bool IsAvailable => File.Exists(CodesignPath);
+
+        public static (int ExitCode, string StdErr) Run(string args, string appHostPath)
+        {
+            Debug.Assert(RuntimeInformation.IsOSPlatform(OSPlatform.OSX));
+            Debug.Assert(IsAvailable);
+
+            var psi = new ProcessStartInfo()
+            {
+                Arguments = $"{args} \"{appHostPath}\"",
```

Comment:
```suggestion
            var psi = new ProcessStartInfo
            {
                Arguments = $"""{args} "{appHostPath}""",
```

Reply:
Ah, right that corner case. (space fixes it but that may not serialize correctly to shell args; multi-line defeats the purpose of it being more succulent 🥲)

Reply:
Ah, right that corner case. (space fixes it but that may not serialize correctly to shell args; multi-line defeats the purpose of it being more succinct 🥲)